package.json

```diff
@@ -29,16 +30,19 @@
     "@radix-ui/react-dropdown-menu": "^2.0.6",
     "@radix-ui/react-icons": "^1.3.0",
     "@radix-ui/react-label": "^2.0.2",
+    "@radix-ui/react-select": "^2.0.0",
     "@radix-ui/react-slot": "^1.0.2",
+    "@radix-ui/react-toast": "^1.1.5",
     "@react-email/components": "0.0.16",
     "@stripe/react-stripe-js": "^2.6.2",
     "@stripe/stripe-js": "^3.2.0",
     "@t3-oss/env-nextjs": "^0.9.2",
+    "@tanstack/react-table": "^8.15.3",
```

Comment:
Why this instead of `@radix-ui/table` ?

Reply:
ReactTable is an headless table/datagrid lib that support paginations, filtering, remote fetching & caching.
We plan to combine it with shadcn/table that is an uncontrolled styled table component